core/platform/datastore/cloud_datastore_services.py

```diff
@@ -147,13 +146,11 @@ def delete_multi(keys):
 
 
 def query_everything(**kwargs):
-    # type: (**Dict[str, Any]) -> ndb.Query
```

Comment:
This file is currently inside mypy denylist. Also it is not yet ready to be fully type checked. The following errors are thrown if I decide to remove it from denylist:
![Screenshot from 2021-08-17 15-16-20](https://user-images.githubusercontent.com/32328861/129703808-572d7986-6ea3-4911-a64b-9ab92a06c970.png)

So I removed unexplained usages of Any, but keeping the other types. When the file will be fully type annotated, this annotation will also get fixed.